media-video/subliminal/subliminal-2.0.5-r3.ebuild

```diff
@@ -0,0 +1,80 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python{2_7,3_5,3_6} )
```

Comment:
Could you test 3.7 while at it?

Reply:
Not right now. I'd need to add 3.7 to `PYTHON_TARGETS` and do a world update and I don't have the time currently.